src/main/proto/netflix/titus/validator/validator.proto

```diff
@@ -158,6 +158,9 @@ message S3BucketAccessValidationResponse {
 
         // True if a bucket can be written to.
         bool writeAccess = 2;
+
+        // In case of access restriction provides more data on why is that.
+        string details = 3;
```

Comment:
If there is an access restriction, wouldn't this be a failure and not a success?

Reply:
This API returns information about access rights. So if we successfully checked that access is denied, it is a valid response, not an error.
If there was however a communication problem with AWS API, and we could not check the access rights, that would be reported as an error.

Reply:
Is this field used only for informational purposes or do we expect response consumer to take action?

Reply:
@joshi-keyur Just information purpose.